celesta-sql/src/test/java/ru/curs/celesta/score/ScoreTest.java

```diff
@@ -323,11 +327,11 @@ public void saveTest() throws ParseException, IOException {
         PrintWriter bw = new PrintWriter(sw);
         new CelestaSerializer(bw).save(t);
         bw.flush();
-        // System.out.println(sw);
+        // LOGGER.info("{}", sw);
```

Comment:
well... do we need this? ) in fact this commented-out lines are just helpers for debugging

recently I learnt that one can set `System.out.println`s  in breakpoint properties in IDEA, without adding `println`s to the code (maybe Eclipse also supports something like this)

so maybe just swipe out the junk from code here....?

Reply:
Done.